src/main/java/com/course/fleet/management/API/controller/UsersController.java

```diff
@@ -0,0 +1,39 @@
+package com.course.fleet.management.API.controller;
+
+import com.course.fleet.management.API.entity.User;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import net.datafaker.Faker;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+@RestController
+@RequestMapping("fleet-management-API/users")
```

Comment:
Hay que buscar la manera de que `fleet-management-API` quedé como el path base para todo tu servicio spring boot. Esto para no tener que agregar esto en cada clase controller.

HINT: Se debe agregar en el application.yml o application.properties, buscalo en google como `context-path`

```suggestion
@RequestMapping("/user")
```